account_invoice_merge/models/account_move.py

```diff
@@ -107,7 +108,11 @@ def make_key(br, fields):
                     or field == "tax_ids"
                     or field == "sale_line_ids"
                 ):
-                    field_val = ((6, 0, tuple(v.id for v in field_val)),)
+                    # field_val = ((6, 0, tuple(v.id for v in field_val)))
+                    field_val = tuple(v.id for v in field_val)
```

Comment:
```python
                    field_val = fields.Commands.set(field_val.ids)
```
according to [odoo documentation](https://www.odoo.com/documentation/16.0/developer/reference/backend/orm.html#odoo.models.Model.write)

Reply:
your suggestion will cause this error :
```
  File "/account-invoicing/16.0/account_invoice_merge/models/account_move.py", line 171, in do_merge
    o_line = invoice_infos["invoice_line_ids"].setdefault(line_key, {})
TypeError: unhashable type: 'list'
```

Reply:
yep, my bad, this should be a list of field.Commands
```suggestion
                    field_val = [fields.Commands.set(field_val.ids)]
```